pkg/controller/azdrivernode.go

```diff
@@ -101,10 +107,18 @@ func (r *ReconcileAzDriverNode) Recover(ctx context.Context) error {
 		if _, err = r.azClient.DiskV1beta2().AzDriverNodes(r.objectNamespace).Update(ctx, updated, metav1.UpdateOptions{}); err != nil {
 			return err
 		}
+		r.AddNodeInNodeDiskAvailabilityMap(ctx, node.Name, node.Labels)
 	}
 	return nil
 }
 
+func (r *ReconcileAzDriverNode) AddNodeInNodeDiskAvailabilityMap(ctx context.Context, nodeName string, nodeLabels map[string]string) {
```

Comment:
We are not using `nodeLabels`. Why pass them?

Reply:
Done.